packages/components/src/AvatarGroup/AvatarGroup.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { HTMLAttributes } from "react"
+import classnames from "classnames"
+import {
+  Avatar,
+  CompanyAvatarProps,
+  GenericAvatarProps,
+} from "~components/Avatar"
+import { OverrideClassName } from "~types/OverrideClassName"
+import styles from "./AvatarGroup.module.scss"
+
+export type AvatarGroupAvatarProps =
+  | Omit<GenericAvatarProps, "size">
+  | Omit<CompanyAvatarProps, "size">
+
+export type AvatarGroupSize = "small" | "medium" | "large"
+export type AvatarList = [AvatarGroupAvatarProps, ...AvatarGroupAvatarProps[]]
+
+export interface AvatarGroupProps
+  extends OverrideClassName<HTMLAttributes<HTMLUListElement>> {
+  /**
+   * There are 3 fixed sizes available. `"small"` will remove border and box shadow to save space.
+   * @default "medium"
+   */
+  size?: AvatarGroupSize
+  /**
+   * If the length of Avatars exceeds the `maxVisible` a counter token will render
+   * @default 2
+   */
+  maxVisible: 2 | 3 | 4 | 5 | 6
+  /**
+   * Takes a array of `AvatarProps` that must have at least item.
+   * Note that 'size' is omitted from the `AvatarProps` type so it will throw a type error if size is provided.
+   * */
+  avatars: AvatarList
+}
```

Comment:
Welcome to the new world! :D

```suggestion
export type AvatarGroupProps = {
  /**
   * There are 3 fixed sizes available. `"small"` will remove border and box shadow to save space.
   * @default "medium"
   */
  size?: AvatarGroupSize
  /**
   * If the length of Avatars exceeds the `maxVisible` a counter token will render
   * @default 2
   */
  maxVisible: 2 | 3 | 4 | 5 | 6
  /**
   * Takes a array of `AvatarProps` that must have at least item.
   * Note that 'size' is omitted from the `AvatarProps` type so it will throw a type error if size is provided.
   * */
  avatars: AvatarList
} & OverrideClassName<HTMLAttributes<HTMLUListElement>>
```

Reply:
You didn't fix this :(